i18n-helpers/src/bin/mdbook-xgettext.rs

```diff
@@ -111,12 +259,34 @@ fn main() -> anyhow::Result<()> {
         .ok_or_else(|| anyhow!("Missing output.xgettext.pot-file config value"))?
```

Comment:
I think we could make things more uniform by removing the `pot-file` config value — replace it by `messages.pot`.

People should then instead use the `-d` or `--dest-dir` argument to `mdbook build` to specify where the POT files should go. So

```
MDBOOK_OUTPUT='{"xgettext": {"pot-file": "messages.pot"}}' mdbook build -d po
```
today becomes

```
MDBOOK_OUTPUT='{"xgettext": {"depth": 0}}' mdbook build -d po
```

(or rather `MDBOOK_OUTPUT='{"xgettext": {}}'` when you make `depth` default to zero).

Reply:
Fixed this in the most recent version - thanks for noting this. I agree it makes more sense without! I've found the directory argument makes most sense now with a `po/XX` (at least in the usage for comprehensive-rust) so all the newly created files can be separated from the language files in the main `po` directory